frontend/src/components/activePage/TableInstance/RowInstanceActions/RowInstanceActionsDefault.tsx

```diff
@@ -65,40 +75,73 @@ const RowInstanceActionsDefault: FC<IRowInstanceActionsDefaultProps> = ({
     ? { href: url!, target: '_blank' }
     : { onClick: () => setSshModal(true), ghost: true };
 
+  const [showDeleteModalConfirm, setShowDeleteModalConfirm] = useState(false);
+
   return (
     <>
-      <Tooltip placement="top" title="Destroy">
-        <Popconfirm
-          placement="left"
-          title="Are you sure to delete?"
-          okText="Yes"
-          cancelText="No"
-          onConfirm={() =>
-            deleteInstanceMutation({
-              variables: {
-                instanceId: name,
-                tenantNamespace: tenantNamespace!,
-              },
-            })
-          }
-          onCancel={e => e?.stopPropagation()}
-        >
+      <ModalAlert
+        headTitle={prettyName}
+        alertMessage={
+          <Text>
+            Delete <b>{name}</b>
+          </Text>
+        }
+        alertDescription={
+          <Text>
+            Do you really want to delete <b>{prettyName}</b> ?
+          </Text>
+        }
```

Comment:
```suggestion
        alertMessage={
          <Text>
            Do you really want to delete <b>{prettyName}</b> ?
          </Text>
        }
        alertDescription=`Instance ID: ${name}`
```
Let's see how it looks like this